static/scss/detail/learning-techniques.scss

```diff
@@ -0,0 +1,75 @@
+// sass-lint:disable mixins-before-declarations
+@import "common";
+
+.learning-techniques-block {
+  padding-top: 70px;
+  padding-bottom: 50px;
+
+  h1 {
+    text-align: center;
+    font-size: 40px;
+    line-height: 44px;
+    margin: 0 0 50px;
+    color: $primary;
+    font-weight: 600;
+    text-transform: uppercase;
+  }
+}
+
+.learning-techniques-list {
+  margin: 0;
+  overflow: hidden;
+  padding: 0;
+  list-style: none;
+  letter-spacing: -.32em;
+  position: relative;
+
+  @include media-breakpoint-up(md) {
+    display: flex;
+    flex-wrap: wrap;
+  }
+
+  li {
+    padding: 20px 35px;
+    margin: 0 0 -1px;
+    letter-spacing: normal;
+    border-bottom: 1px solid $border-top;
+    text-align: center;
+    color: black;
+
+    @include media-breakpoint-up(md) {
+      margin: 0 0 -1px -1px;
+      width: 50%;
+      display: inline-block;
+      vertical-align: top;
```

Comment:
In testing this, it looks like both `display: inline-block;` and `vertical-align: top;` are unnecessary